### PR TITLE
Python script to send SMS

### PR DESCRIPTION
**Pull request by soumya-ranjan7**
A simple python script to send SMS.

### CODE REVIEW

**Review by amitsagtani97 — CHANGES_REQUESTED** [review]
can you please create a new directory and move this file into that directory with a README file :)

**Comment by soumya-ranjan7** [comment]
Requested changes have been made.

File: Sending_SMS/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This a simple python script to send SMS to any desired phone number through Sinch messaging service (free of cost).
```
Comment (amitsagtani97) [inline Sending_SMS/README.md:1]: @soumya-ranjan7 Can you please add a link to Sinch API, where user can sign up and get a API Key to use your script.

**Comment by soumya-ranjan7** [comment]
Link to Sinch SMS API has been added.

**Comment by amitsagtani97** [comment]
@soumya-ranjan7  thanks for the PR. :) :+1:

**Pull request merged**